--- generic/jkFormatMP3.c.orig	Thu Apr 26 09:31:49 2007
+++ generic/jkFormatMP3.c	Thu Apr 26 09:32:36 2007
@@ -31,7 +31,7 @@
 #include <string.h>
 #define FRAS2(is,a) ((is) > 0 ? t_43[(is)]*(a):-t_43[-(is)]*(a))
 #define MAXFRAMESIZE 2106  /* frame size starting at header */
-#define roundf(x) (floor((x)+(float )0.5f))
+//#define roundf(x) (floor((x)+(float )0.5f))
 static char *gblOutputbuf;
 static char *gblReadbuf;
 static int gblBufind = 0;
--- generic/jkSound.h.orig	2014-02-13 12:53:58 +0400
+++ generic/jkSound.h	2014-02-13 12:54:05 +0400
@@ -603,7 +603,7 @@
 
 extern void SnackPauseAudio();
 
-#if TCL_MAJOR_VERSION == 8 && TCL_MINOR_VERSION < 4
+#if TCL_MAJOR_VERSION == 8 && TCL_MINOR_VERSION < 7
 #define TCL_SEEK Tcl_Seek
 #define TCL_TELL Tcl_Tell
 #else

--- win/jkAudIO_win.c.orig  2005-12-14 14:29:38 +0300
+++ win/jkAudIO_win.c       2020-01-16 15:38:19 +0300
@@ -58,15 +58,6 @@
 /*static ADesc  winad;*/
 static int    correction = 1;

-const static GUID  KSDATAFORMAT_SUBTYPE_PCM = {0x00000001,0x0000,0x0010,
-                                                {0x80,
-                                                0x00,
-                                                0x00,
-                                                0xaa,
-                                                0x00,
-                                                0x38,
-                                                0x9b,
-                                                0x71}};
 #define SNACK_NUMBER_MIXERS 1

 struct MixerLink mixerLinks[SNACK_NUMBER_MIXERS][2];
--- win/configure.in.orig	2020-01-20 17:27:33.516586100 +0300
+++ win/configure.in	2020-01-20 17:28:46.438820000 +0300
@@ -5,7 +5,7 @@
 AC_INIT(../generic/snack.h)
 
-SNACK_VERSION=`sh $srcdir/sver.sh $srcdir/../generic/snack.h`
+SNACK_VERSION=2.2
-SNACK_NODOT_VERSION=`echo $SNACK_VERSION | tr -d .`
+SNACK_NODOT_VERSION=22
 
 AC_CHECK_HEADER(stdint.h)
 
 
--- win/Makefile.in.orig	2005-12-14 14:29:39.000000000 +0300
+++ win/Makefile.in	2020-01-20 17:45:20.624981100 +0300
@@ -1,7 +1,7 @@
 srcdir = @srcdir@
 prefix = @prefix@
 
-CC = gcc
+CC = @CC@
 CFLAGS = @CFLAGS@
 
 INSTALL_DIR = @prefix@
@@ -121,10 +121,10 @@
 	cp $(SNACK_PKG_FILE) "$(SNACK_INSTALL_DIR)/$(SNACK_PKG_FILE_DEST)"
 
 $(SNACK_DLL): $(SNACK_OBJS)
-	gcc -shared -o $@ $(SNACK_OBJS) $(SNACK_LIBS)
+	@CC@ -shared -o $@ $(LDFLAGS) $(SNACK_OBJS) $(SNACK_LIBS)
 
 $(SOUND_DLL): $(SOUND_OBJS)
-	gcc -shared -o $@ $(SOUND_OBJS) $(SOUND_LIBS)
+	@CC@ -shared -o $@ $(LDFLAGS) $(SOUND_OBJS) $(SOUND_LIBS)
 
 $(SNACK_STUB) : $(SNACK_STUB_OBJS)
 	ar cr $@ $(SNACK_STUB_OBJS)
