--- win/Makefile.in.orig	2019-11-21 22:10:50 +0300
+++ win/Makefile.in	2020-01-16 17:48:33 +0300
@@ -97,7 +97,7 @@
 
 SRC_DIR			= @srcdir@
 ROOT_DIR		= @srcdir@/..
-TOP_DIR			= $(shell cd @srcdir@/..; pwd -W 2>/dev/null || pwd -P)
+TOP_DIR			= $(shell cd @srcdir@/..; pwd -P)
 GENERIC_DIR		= $(TOP_DIR)/generic
 TOMMATH_DIR		= $(TOP_DIR)/libtommath
 WIN_DIR			= $(TOP_DIR)/win
--- win/tcl.m4.orig	2019-11-21 22:10:50 +0300
+++ win/tcl.m4	2020-01-16 17:55:38 +0300
@@ -679,7 +679,7 @@
 	STLIB_LD='${AR} cr'
 	RC_OUT=-o
 	RC_TYPE=
-	RC_INCLUDE=--include
+	RC_INCLUDE=--include-dir
 	RC_DEFINE=--define
 	RES=res.o
 	MAKE_LIB="\${STLIB_LD} \[$]@"
