--- configure.in.orig	2012-07-10 03:28:02 +0400
+++ configure.in	2014-02-13 15:37:23 +0400
@@ -164,25 +164,6 @@
 
 TEA_MAKE_LIB
 
-#--------------------------------------------------------------------
-# Shared libraries and static libraries have different names.
-# Also, windows libraries and unix libraries have different names.
-# For the OpenSSL version, I chose to use the same library names that
-# OpenSSL uses as its default names.
-#--------------------------------------------------------------------
-
-if test "${TEA_PLATFORM}" = "windows" ; then
-    if test "$GCC" = "yes"; then
-	TEA_ADD_LIBS([-L${SSL_LIB_DIR_NATIVE}])
-    else
-	TEA_ADD_LIBS([-libpath:${SSL_LIB_DIR_NATIVE}])
-    fi
-    if test -n "${OPENSSL}"; then
-        TEA_ADD_LIBS([ssleay32.lib libeay32.lib])
-    else
-        TEA_ADD_LIBS([sslc32.lib])
-    fi
-else
     # Subst runtime dir here, use -R and -L where necessary. [Bug 1742859]
     LIB_RUNTIME_DIR=${SSL_LIB_DIR}
     eval "LD_SEARCH_FLAGS=\"${LD_SEARCH_FLAGS}\""
@@ -191,7 +172,6 @@
     else
 	TEA_ADD_LIBS([${LD_SEARCH_FLAGS} -L${SSL_LIB_DIR} -lsslc])
     fi
-fi
 
 #--------------------------------------------------------------------
 # Find tclsh so that we can run pkg_mkIndex to generate the pkgIndex.tcl
